irPassLLVMIRAutoQuantization causing build fail

**Describe the bug**
There is a build failure because of the undefined function `irPassLLVMIRAutoQuantization` in `newton.c`. The corresponding header file for the quantization pass is not included in `newton.c`. At the moment, including this file leads to further build errors that I will detail in a follow up issue.

```
newton.c:215:3: error: call to undeclared function 'irPassLLVMIRAutoQuantization'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
                irPassLLVMIRAutoQuantization(N);
```

**To Reproduce**
Steps to reproduce the behavior:
1. Clone and prepare for build
2. Attempt `make`
3. See error.

**Expected behavior**
Compilation should complete successfully.

**Host OS (please complete the following information):**
 - OS: macos 14.5

--- a/src/newton/main.c
+++ b/src/newton/main.c
@@ -107,9 +107,9 @@ main(int argc, char *argv[])
 			{"latex",		no_argument,		0,	'x'},
 			{"RTLcodegen",		required_argument,	0,	'l'},
 			{"targetParam",		required_argument,	0,	'T'},
-			{"llvm-ir",             required_argument,      0,      'I'},
-			{"llvm-ir-liveness-check",    no_argument,      0,      'L'},
-            {"llvm-ir-auto-quantization",    no_argument,      0,      'Q'},
+			{"llvm-ir",		required_argument,	0,	'I'},
+			{"llvm-ir-liveness-check",	no_argument,	0,	'L'},
+			{"llvm-ir-auto-quantization",	no_argument,	0,	'Q'},
 			{"estimator-synthesis",	required_argument,	0,	420},
 			{"process",		required_argument,	0,	421},
 			{"measurement",		required_argument,	0,	422},
@@ -424,11 +424,16 @@ main(int argc, char *argv[])
 				break;
 			}
 
-            case 'Q':
-            {
-                N->irPasses |= kNewtonirPassLLVMIRAutoQuantization;
-                break;
-            }
+			case 'Q':
+			{
+				N->irPasses |= kNewtonirPassLLVMIRAutoQuantization;
+
+				flexprint(N->Fe, N->Fm, N->Fperr, "AutoQuantization pass was disabled at compilation.\n");
+				consolePrintBuffers(N);
+				exit(EXIT_FAILURE);
+
+				break;
+			}
 
 			case 494:
 			{

--- a/src/newton/newton.c
+++ b/src/newton/newton.c
@@ -206,14 +206,14 @@ processNewtonFile(State *  N, char *  filename)
 	{
 		irPassLLVMIROptimizeByRange(N);
 	}
-
-	/*
-	 *	irPassLLVMIRAutoQuantization pass is causing issues with the current implementation.
-	 */
-	// if (N->irPasses & kNewtonirPassLLVMIRAutoQuantization)
-	// {
-	// 	irPassLLVMIRAutoQuantization(N);
-	// }
+	if (N->irPasses & kNewtonirPassLLVMIRAutoQuantization)
+	{
+		/*
+		 *	irPassLLVMIRAutoQuantization pass is causing issues with the current implementation.
+		 */
+		// irPassLLVMIRAutoQuantization(N);
+		flexprint(N->Fe, N->Fm, N->Fperr, "AutoQuantization pass was disabled at compilation.\n");
+	}
 
 	/*
 	 *	Dot backend.

--- a/src/newton/newton.c
+++ b/src/newton/newton.c
@@ -206,10 +206,15 @@ processNewtonFile(State *  N, char *  filename)
 	{
 		irPassLLVMIROptimizeByRange(N);
 	}
-    if (N->irPasses & kNewtonirPassLLVMIRAutoQuantization)
-    {
-        irPassLLVMIRAutoQuantization(N);
-    }
+
+	/*
+	 *	irPassLLVMIRAutoQuantization pass is causing issues with the current implementation.
+	 */
+	// if (N->irPasses & kNewtonirPassLLVMIRAutoQuantization)
+	// {
+	// 	irPassLLVMIRAutoQuantization(N);
+	// }
+
 	/*
 	 *	Dot backend.
 	 */

